docs/user/customization.md

```diff
@@ -158,6 +161,8 @@ Cursorless exposes a couple talon actions and captures that you can use to defin
 - `user.cursorless_insert(destination: CursorlessDestination, text: Union[str, List[str]])`:
   Insert text at destination.
   eg: `user.cursorless_insert(cursorless_destination, "hello")`
+- `user.cursorless_create_destination(target: CursorlessTarget, insertion_mode: Literal["to", "before", "after"] = "to")`:
```

Comment:
Do we want to add the return type to the function signature?

Reply:
Idk the line is pretty long as is, but happy to if you prefer

Reply:
I mean since it's actually returning something important I would prefer that. We do have return type on the get text action for example.

Reply:
Sounds good will add